tools/pg_inspector/active_connections_to_human.rb

```diff
@@ -136,15 +144,21 @@ def push_other_process(result, activity)
     end
 
     def server_activity?(activity)
-      miq_activity?(activity) && activity["application_name"].split("|")[3] == "-"
+      if activity["application_name"].start_with?("MIQ|") && activity["application_name"].split("|")[3] == "-"
+        return true
+      elsif activity["application_name"].start_with?("MIQ ") && activity["application_name"].include?(" Server")
```

Comment:
We have this conditional check "is this the new format or old format?" in a few places in this PR, maybe we can put that in a method after this is merged.

Reply:
Right, will do that.